CHANGE_LOG

```diff
@@ -1,3 +1,423 @@
+* PR `#8788 <https://github.com/numba/numba/pull/8788>`_: CUDA: Fix returned dtype of vectorized functions (Issue #8400) (`gmarkall <https://github.com/gmarkall>`_)
+* PR `#8790 <https://github.com/numba/numba/pull/8790>`_: CUDA compare and swap with index (`ianthomas23 <https://github.com/ianthomas23>`_)
+* PR `#8795 <https://github.com/numba/numba/pull/8795>`_: Add pending-deprecation warnings for ``numba.pycc`` (`stuartarchibald <https://github.com/stuartarchibald>`_)
+* PR `#8802 <https://github.com/numba/numba/pull/8802>`_: Move the minimum supported NumPy version to 1.21 (`stuartarchibald <https://github.com/stuartarchibald>`_)
+* PR `#8803 <https://github.com/numba/numba/pull/8803>`_: Attempted fix to #8789 by changing `compile_ptx` to accept a signature instead of argument tuple (`KyanCheung <https://github.com/KyanCheung>`_)
+* PR `#8804 <https://github.com/numba/numba/pull/8804>`_: Split parfor pass into 3 parts (`DrTodd13 <https://github.com/DrTodd13>`_)
+* PR `#8809 <https://github.com/numba/numba/pull/8809>`_: Update LLVM versions for 0.57 release (`apmasell <https://github.com/apmasell>`_)
+* PR `#8810 <https://github.com/numba/numba/pull/8810>`_: Fix llvmlite dependency in meta.yaml (`sklam <https://github.com/sklam>`_)
+* PR `#8816 <https://github.com/numba/numba/pull/8816>`_: Fix some buildfarm test failures (`sklam <https://github.com/sklam>`_)
+* PR `#8819 <https://github.com/numba/numba/pull/8819>`_: Support "static" __getitem__ on Numba types in `@njit` code. (`stuartarchibald <https://github.com/stuartarchibald>`_)
+* PR `#8822 <https://github.com/numba/numba/pull/8822>`_: Merge py3.11 branch to main (`esc <https://github.com/esc>`_ `AndrewVallette <https://github.com/AndrewVallette>`_ `stuartarchibald <https://github.com/stuartarchibald>`_ `sklam <https://github.com/sklam>`_)
+* PR `#8826 <https://github.com/numba/numba/pull/8826>`_: CUDA CFFI test: conditionally require cffi module (`gmarkall <https://github.com/gmarkall>`_)
+* PR `#8831 <https://github.com/numba/numba/pull/8831>`_: Redo py3.11 sync branch with main  (`dlee992 <https://github.com/dlee992>`_ `bszollosinagy <https://github.com/bszollosinagy>`_ `stuartarchibald <https://github.com/stuartarchibald>`_ `neilflood <https://github.com/neilflood>`_ `brandonwillard <https://github.com/brandonwillard>`_ `MiloniAtal <https://github.com/MiloniAtal>`_ `luk-f-a <https://github.com/luk-f-a>`_ `thomasjpfan <https://github.com/thomasjpfan>`_ `alanhdu <https://github.com/alanhdu>`_ `guilhermeleobas <https://github.com/guilhermeleobas>`_ `fangchenli <https://github.com/fangchenli>`_ `gmarkall <https://github.com/gmarkall>`_ `ianthomas23 <https://github.com/ianthomas23>`_ `sklam <https://github.com/sklam>`_ `apmasell <https://github.com/apmasell>`_ `kc611 <https://github.com/kc611>`_ `DrTodd13 <https://github.com/DrTodd13>`_ `esc <https://github.com/esc>`_ `testhound <https://github.com/testhound>`_ `oscargus <https://github.com/oscargus>`_)
```

Comment:
Am not sure what to do with the attribution here, it's auto-generated and "correct" given what was asked of it. The authors list is extensive as a result of merging `main` into the Python 3.11 feature branch and then making a PR for that so as to merge it into `main`.

Reply:
Since this PR is just me doing the merge and all other py3.11 branch contribution has been accounted for in other PRs, the blame list should just include me.

Reply:
Addressed in 5ff02fe.